### DISCUSSION TITLE
Let's get "Galaxy brain achievement"

### DISCUSSION THREAD

**Discussion by fynntang** [Schweinepriester/github-profile-achievements#89] [Q&A]
> Everyone initiates two Q&A then others reply, and then the initiator chooses a satisfactory reply

**Play:**
Everyone replies their Q&A links to this post, and then attaches the corresponding answers, and then everyone works together to achieve success

**Note:** 
you should give priority to small partners who have not yet achieved

**Comment by ddh** [comment]
Yes, let's work together to get our answers marked satisfactory 👍🏼
